src/middleware/auth.js

```diff
@@ -315,11 +316,11 @@ const remoteAuthentication = ({signatureValidator, entry}) => {
       }
     } catch (err) {
       if (err.code === 'AuthorizationError') {
-        return res.replyError('InsufficientScopes', err.messageTemplate, err.details);
+        return next(new ErrorReply({code: 'InsufficientScopes', message: err.messageTemplate, details: err.details}));
       } else if (err.code === 'AuthenticationError') {
-        return res.replyError('AuthenticationFailed', err.message, err.details);
+        return next(new ErrorReply({code: 'AuthenticationFailed', message: err.message, details: err.details}));
       }
-      return res.replyError(err);
+      return next(new ErrorReply(err));
```

Comment:
This one is *not* an ErrorReply -- we already have an error that we need to handle.  So just `next(err)`.